ansible_collections/arista/avd/roles/eos_designs/doc/network-services.md

```diff
@@ -138,6 +142,26 @@ tenants:
             - pod: < pod_name >
               ipv4_pool: < IPv4_address/Mask >
 
+        # Dictionary for router OSPF configuration | Optional.
+        # This will create an ospf routing instance in the tenant VRF. If there is no nodes definition, the ospf instance will be
+        # created on all leafs where the vrf is deployed. This will also cause automatic ospf redistribution into bgp unless
+        # explicitly turned off with "redistribute_ospf: false".
+        ospf:
+          enabled: < true | false >
+          process_id: < int, Default -> vrf_id >
+          router_id: < router-id, Default -> switch router_id >
+          max_lsa: < int >
+          bfd: < true | false, Default -> false >
+          redistribute_bgp:
+            enabled: < true | false >
+            route_map: < route-map name >
+          nodes:
+            - < hostname1 >
+            - < hostname2 >
+
+        # Non-selectively enabling or disabling redistribute ospf inside the VRF | Optional.
+        redistribute_ospf: < true | false >
```

Comment:
The logic described in J2 template suggests this value is set to True by default.

Reply:
Updated documentation accordingly.